src/main/java/com/synopsys/integration/jenkins/detect/substeps/DetectJarManager.java

```diff
@@ -77,15 +73,15 @@ public String calculateJavaExecutablePath() throws IOException {
         return javaExecutablePath;
     }
 
-    public void logJavaVersion() throws InterruptedException {
-        logger.debug("PATH: " + environmentVariables.get("PATH"));
-        if (LogLevel.DEBUG == logger.getLogLevel()) {
+    private void logDebugData(String javaExecutablePath) throws InterruptedException {
+        if (logger.getLogLevel().isLoggable(LogLevel.DEBUG)) {
```

Comment:
The loglevel will be INFO if not specifically set. If, for some reason, the setLogLevel(null) is run, the logger.info lines above will fail with a NullPointerException.